spec/rails_helper.rb

```diff
@@ -1,64 +1,32 @@
-# This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
```

Comment:
Đổi hết sang nháy kép trong file .rb bạn nhé
```suggestion
require "spec_helper"
```